src/webview/contextMenuBuilder.ts

```diff
@@ -633,27 +633,45 @@ export class ContextMenuBuilder {
     menu.append(copyImageUrl);
 
     // TODO: This doesn't seem to work on linux, so, limiting to Mac for now
```

Comment:
Are you planning to make the changes cross-platform compatible?

Reply:
I haven't made changes in this PR to expand the compatibility across OSes @exilvm . I also do not have access to a linux box to test any changes.
Finally, I'm moving off of Ferdium for the foreseeable future - so, will request other contributors to take this PR up.

Reply:
Since this comment is not for changes relevant to this PR, I'm closing this comment